lesson_13/homework_13_1/homework_13_1.py

```diff
@@ -0,0 +1,14 @@
+
+with open('result_kovaliuk.csv', 'w') as response_file:
```

Comment:
It is better to divide it into several or at least 1 function instead of only with open syntax construction.